src/providers/aws.rs

```diff
@@ -233,10 +230,7 @@ where
         let prov = self.get_credential_provider()?;
         let mut awsregion = AWSRegion::new(&l.region.to_string(), prov, l.log)?;
         awsregion.make_spot_instance_requests(
-            std::cmp::min(
-                360,
-                (self.max_instance_duration.as_secs() as i64 / 60) as i64,
-            ),
+            std::cmp::min(360, self.max_instance_duration_hours * 60),
```

Comment:
Shouldn't this be `* 3600`?

Reply:
no? x hours * 60 minutes/hour

Reply:
I will add a comment

Reply:
I thought the API took the duration in seconds?

Reply:
minutes: https://docs.rs/rusoto_ec2/0.41.0/rusoto_ec2/struct.RequestSpotInstancesRequest.html#structfield.block_duration_minutes